### PR TITLE
added api for fetching tokentx on fuse

### PR DESCRIPTION
**Pull request by LiorAgnin**
subj

### CODE REVIEW

File: server/src/tasks/relay.js [hunk 1]
```diff
@@ -168,6 +172,8 @@ const relay = withWalletAccount(async (account, { walletAddress, methodName, met
         job.save()
       }
     } else {
+      job.attrs.data.status = 'failed'
```
Comment (leonprou) [inline server/src/tasks/relay.js:175]: there should be blockNumber for failing as well I think

File: server/src/routes/api/v2/wallets/transfers.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  const { walletAddress } = req.params
+  const { tokenAddress, sort = 'desc', startblock = 0 } = req.query
+  const responseTransferEvents = await request.get(`${config.get('explorer.fuse.urlBase')}?module=account&action=tokentx&contractaddress=${tokenAddress}&address=${walletAddress}&startblock=${startblock}&sort=${sort}`)
+  const transferEvents = JSON.parse(responseTransferEvents)
+
+  if (transferEvents['status'] === '1') {
+    const result = await Promise.map(transferEvents['result'], transferEvent => {
+      return new Promise(resolve => {
+        if (transferEvent.from.toLowerCase() === walletAddress.toLowerCase()) {
+          AgendaJob.findOne({
```
Comment (leonprou) [inline server/src/routes/api/v2/wallets/transfers.js:46]: We can do this later in one query with multiple hashes to optimize this

File: server/.gitignore [hunk 3]
```diff
@@ -35,6 +35,7 @@ node_modules
 .node_repl_history
 
 .idea
+run.sh
```
Comment (leonprou) [inline server/.gitignore:38]: what's that :) ?

File: server/src/tasks/relay.js (hunk 1 above)
Comment (LiorAgnin) [reply server/src/tasks/relay.js:175]: you right, I fixed that.

File: server/.gitignore (hunk 3 above)
Comment (LiorAgnin) [reply server/.gitignore:38]: oops

**Pull request merged**